### PR TITLE
fix fortress docs

### PR DESCRIPTION
**Pull request by khaledgabr77**
1. Changed the linear velocity from `0.5` to `-0.5` in downward movement line. 

 *  **Before**:
    * Down &#10142; 16777237 &#10142; linear: {x: 0.5}, angular: {z: 0.0}
  
  *  **After**:
     * Down &#10142; 16777237 &#10142; linear: {x: -0.5}, angular: {z: 0.0}

2. Changed the file path for `rrbot.urdf` in the `spawn_urdf` tutorial.

### CODE REVIEW

File: fortress/spawn_urdf.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ More information about URDF can be found at [https://wiki.ros.org/urdf](https://
 ## Obtaining a URDF file
 
 This tutorial assumes that the reader already has a URDF file that they'd like to load into Gazebo Sim.
-If you are looking for a URDF file to use, you can use the [`rrbot.urdf`](tutorials/spawn_urdf/rrbot.urdf) that was preprocessed for you. The `rrbot` robot model is from the [`gazebo_ros_demos`](https://github.com/ros-simulation/gazebo_ros_demos) package. You could also use one of the examples listed at [https://wiki.ros.org/urdf/Examples](https://wiki.ros.org/urdf/Examples).
+If you are looking for a URDF file to use, you can use the [`rrbot.urdf`](https://github.com/khaledgabr77/docs/blob/96bd6602bb58c9a815519580a713b050634e6e60/fortress/tutorials/spawn_urdf/rrbot.urdf) that was preprocessed for you. The `rrbot` robot model is from the [`gazebo_ros_demos`](https://github.com/ros-simulation/gazebo_ros_demos) package. You could also use one of the examples listed at [https://wiki.ros.org/urdf/Examples](https://wiki.ros.org/urdf/Examples).
```
Comment (azeey) [inline fortress/spawn_urdf.md:11]: To fix the path on Fortress, I think we need to copy `rrbot.urdf` to the `fortress` folder instead of this change here.

**Review by azeey — CHANGES_REQUESTED** [review]
Thanks for the contribution!

File: fortress/spawn_urdf.md (hunk 1 above)
Comment (khaledgabr77) [reply fortress/spawn_urdf.md]: Ah, got it! Moving `rrbot.urdf` to the fortress folder does make more sense. I'll go ahead and do that instead. Thanks for pointing it out!

Comment (azeey) [reply fortress/spawn_urdf.md]: Do you need to `git add` the file? I don't see it in the PR.

Comment (khaledgabr77) [reply fortress/spawn_urdf.md]: You can navigate below the comments to locate the commit titled [update rrbot.urdf path](https://github.com/gazebosim/docs/pull/423/commits/b8721a6563f7a6b80ff461c47693961182fad5f6).

**Review by azeey — APPROVED** [review]

**Pull request merged**